plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/DeltaLakeParquetStatisticsUtils.java

```diff
@@ -71,6 +92,123 @@ public static boolean hasInvalidStatistics(Collection<ColumnChunkMetaData> metad
                         (!metadata.getStatistics().hasNonNullValue() && metadata.getStatistics().getNumNulls() != metadata.getValueCount()));
     }
 
+    @Nullable
+    public static Object toTrinoValue(Type type, @Nullable Object value)
```

Comment:
Can we say what Object representation we're translating from? Either in the method name or in a comment.